### DISCUSSION TITLE
problem in breeze turbo mode .

### DISCUSSION THREAD

**Discussion by ansarhusain7** [breezefront/community#284] [3. Integrations]
Hello @vovayatsyuk ,

I have made breeze compatible module which is not working in turbo mode.

Can you help me please what i need to do for turbo mode ?

Thanks

**Comment by vovayatsyuk** [comment]
It depends on how your current code looks like. Do you have a link to your site?

**Reply by ansarhusain7** [reply]
Here you can see @vovayatsyuk 

https://breeze.uandi.work/driven-backpack

after add to cart , click on edit cart in mini cart it will show all options because not load js 

Thanks

**Reply by vovayatsyuk** [reply]
When I add the item to the cart, it redirects me to the shopping cart.
I'm not sure I understand how it's related to the Turbo.

**Reply by ansarhusain7** [reply]
Hello @vovayatsyuk ,

can you look video here you will understand issue ?

https://www.loom.com/share/6118f06701004f2cbae2a029060abac3

Thanks

**Reply by vovayatsyuk** [reply]
Try moving all of your props into options:

```js
$.widget('pektsekye.optionExtended', {
    component: 'Pektsekye_OptionExtended/js/widget',
    options: {
        dependecyIsSet: false,
        v: [],
        o: [],
        oldO: [],
        imageLayerIds: [],
        templatePattern: /(^|.|\r|\n)({{(\w+)}})/,
        oIds: [],
        oldV: [],
        oldO: [],
        childrenVals: [],
        indByValue: [],
        valsByOption: [],
        optionByValue: [],
        univValsByOption: [],
        childOIdsByO: [],
        previousIds: [],
        childrenByOption: [],
    },
    _create: function() {
        $.extend(this, this.options);
```
